README.md

```diff
@@ -69,25 +71,23 @@ The User data is synced between the different connected devices only when the Us
 #### `masq.connectToMasq() -> Promise<void>`
 
 Connects to Masq if a User is already logged into Masq.
-The data will then be synced between the different connected devices of the User
+The data will then be synced between the different connected devices of the User.
 
 #### `masq.signout() -> Promise<void>`
 
 Signs out of Masq from Masq.
 This also means the User-app data on the device will stop being synced.
 
-#### `masq.logIntoMasq(stayConnected) -> Promise<ret: object>`
-
-`stayConnected` is a boolean representing if the User should stay connected until he explicitely signs out.
+### `masq.getLoginLink() -> Promise<link: string>`
 
-The function returns an object containing an attribute `link` containing the Url the user has to access to connect to Masq.
+The function returns a Promise resolving to an Url the user has to open to connect to Masq.
```

Comment:
Proposition: The function returns a Promise resolving to an Url, which must be opened by the user in order to connect to Masq.

Reply:
changed !